### PR TITLE
Create Enum.md

### PR DESCRIPTION
**Pull request by NiranjanNlc**
Added the description about the  concept of enum in Kotlin

### CODE REVIEW

**Comment by NiranjanNlc** [comment]
closes #653

File: Kotlin/Enum.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+val redColor = Color.RED
+println(redColor.rgb)           // Output: 16711680
+println(redColor.description()) // Output: Red color
+```
+
+Enums in Kotlin can be useful in various scenarios, such as representing days of the week, menu options, status codes, and more. They provide a type-safe and concise way to define a fixed set of values.
+
+For more information about enums in Kotlin, you can refer to the following resources:
+
+- [Kotlin Enum Classes](https://kotlinlang.org/docs/enum-classes.html)
```
Comment (Kirchhoff-) [inline Kotlin/Enum.md:49]: Could you please create the section "Further Reading" (like here - [link](https://github.com/Kirchhoff-/Android-Interview-Questions/blob/master/Android/What%20do%20you%20know%20about%20Auto%20Backup.md#further-reading)) and move those two links to newly section.

**Comment by Kirchhoff-** [comment]
Hi @NiranjanNlc . Sorry for the late response, was a little bit busy at work. 

First of all, I wanna say thank you for your help. I left one comment that need to be fixed before I can merge your work. And also, please increment kotlin answer counter in [`README.md`](https://github.com/Kirchhoff-/Android-Interview-Questions/blob/1fca1d0b72769adbfb96a6823848e8a2854b93a0/README.md) file. 

Thanks!!

**Comment by NiranjanNlc** [comment]
I think we are good to go now .

**Pull request merged**